eZ/Publish/API/Repository/Tests/Values/User/Limitation/LanguageLimitationTest.php

```diff
@@ -650,12 +650,13 @@ public function testHideLocationWithLanguageLimitationAndDifferentContentTransla
      * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
      * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
      */
-    public function testHideAndUnhideLocationWithLanguageLimitationAndDifferentContentTranslations(
+    public function testHideAndUnhideLocationAndContentWithLanguageLimitationAndDifferentContentTranslations(
```

Comment:
Too much happens in this test now, it should be split. If you want to use previous test return value you can use `@depends`.

Reply:
I cannot use `@depends` as a repository is cleaned after each use.